setup.py

```diff
@@ -39,7 +39,7 @@
     long_description_content_type="text/markdown",
     url=metadata["__url__"],
     packages=setuptools.find_packages(),
-    python_requires=">=3.8",
+    python_requires=">=3.11",
```

Comment:
Does it work with Python 3.13 yet? Should we make that explicit here?

Reply:
It doesn't. I added that that chec.